pages/confirmation/[wager].js

```diff
@@ -0,0 +1,58 @@
+import { styles } from '../../constants/styles';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Wrapper from '../../components/Wrapper';
+
+const data = {
+  description: 'Make money while putting your intuition on the line.',
+  header: 'Your Submission been received!',
+};
+const Confirmation = ({ wager }) => (
+  <div>
+    <Wrapper data={data} className='measure-wide'>
+      <section>
+        <p className={`${styles.paragraph}`}>
+          Complete the final step to complete your wager{' '}
```

Comment:
remove `{' '}` in files

Reply:
i have to bro. otherwise theres no space between that last word and the word in the <a>